create_jenkins_job.py

```diff
@@ -73,12 +73,11 @@ def main(argv=None):
         'supplemental_repos_url': '',
         'time_trigger_spec': '',
         'mailer_recipients': '',
-        'use_connext_default': 'true',
-        'disable_connext_static_default': 'false',
-        'disable_connext_dynamic_default': 'true',
+        'ignore_rmw_default': {
+            'rmw_connext_dynamic_cpp',
+            'rmw_fastrtps_dynamic_cpp',
+            'rmw_opensplice_cpp'},
```

Comment:
The dynamic implementations of Connext and FastRTPS as well as OpenSplice are ignored by default